gemfirexd/tools/src/main/java/com/pivotal/gemfirexd/internal/impl/tools/ij/utilMain.java

```diff
@@ -765,8 +765,15 @@ private boolean doCatch(String command) {
                         else {
                           redirected = null;
                         }
+// GemStone changes BEGIN
+      boolean displayCount = false;
+      if (command.startsWith("insert") || command.startsWith("update")
+        || command.startsWith("delete") || command.startsWith("put") ) {
+        displayCount = true;
+      }
```

Comment:
Changed it. But there already is a code which uses this to find out whether the firs one is 'set' or 'elapsed'. I will change them too.